src/auth/auth.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Body, Post, HttpCode, HttpStatus } from '@nestjs/common';
+import { AuthService } from './auth.service';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @HttpCode(HttpStatus.OK)
+  @Post('login')
+  async signIn(@Body() userData: Record<string, any>) {
```

Comment:
in routes like this try to keep consistency between route / controller naming. Here we have a `login` word in route. In this case the controller method should be `login`

Reply:
you can also change the route to `signin`